fixtures/prolights/smartbat.json

```
@@ -0,0 +1,270 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "SMARTBAT",
+  "categories": ["Color Changer"],
+  "meta": {
+    "authors": ["Matteo Piccina"],
+    "createDate": "2019-12-23",
+    "lastModifyDate": "2019-12-23"
+  },
+  "links": {
+    "manual": [
+      "https://www.musiclights.it/download.php?id=3844"
+    ],
+    "productPage": [
+      "https://www.musiclights.it/product/SMARTBAT"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=2A7gp6_6tMQ"
+    ]
+  },
+  "physical": {
+    "dimensions": [137.8, 184, 145],
+    "weight": 3.4,
+    "power": 30,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "4× 8W RGBW LED",
+      "lumens": 1127
+    },
+    "lens": {
+      "degreesMinMax": [15, 15]
+    }
+  },
+  "availableChannels": {
+    "Dimmer": {
+      "capability": {
+        "type": "Intensity"
+      }
+    },
+    "Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Color Presets / Temperature": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 10],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [11, 30],
+          "type": "ColorPreset",
+          "comment": "Red … Green",
```

Revert my own suggestion:

```suggestion
          "comment": "Red … Yellow",
```